flake8diff/flake8.py

```diff
@@ -129,7 +129,7 @@ def get_vcs(self):
             try:
                 vcs = vcs(self.commits, self.options)
             except VCSNotInstalledError:
-                logger.error('Seems like {0} is not installed'.format(vcs.name))
+                logger.info('Seems like {0} is not installed'.format(vcs.name))
```

Comment:
this now will only print in console when verbose flag is used. otherwise it silently fails and proceeds to check other VCS. if none are found, then it flake8-diff will exit with error